eureka2-server/src/main/java/com/netflix/eureka2/server/channel/InterestNotificationMultiplexer.java

```diff
@@ -112,6 +112,6 @@ public void unregister() {
      * simplifies implementation of the multiplexer.
      */
     public Observable<ChangeNotification<InstanceInfo>> changeNotifications() {
-        return aggregatedStream;
+        return aggregatedStream.share();
```

Comment:
There is no need for that as channel contract guarantees single subscription only.